### PR TITLE
Milestone 2: Add setup and mobile first - portfolio

### PR DESCRIPTION
**Pull request by yemidada**
In this milestone, I:
- Learn how to translate a Figma design into a functional UI.
- Utilize Flexbox to position various elements on a web page.
- Develop a portfolio website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <header>
+        <ul class="menu">
+            <li class="logo"><a href="#" class="logo_link link text-center fz-18 fw-700 lh-20">My logo</a></li>
+            <li class="item"><a href="#">Portfolio</a></li>
+            <li class="item"><a href="#">About</a></li>
+            <li class="item"><a href="#">Contact</a></li>
+            <li class="mobile-menu"><a href="#">
+                    <img src="assets/images/header-menu.svg" alt="Hamburguer Icon">
+                </a></li>
+        </ul>
```
Comment (iambenkis) [inline index.html:23]: Good job adding the hamburger button, 👏  ,  noticed that the menu list and the hamburger button should be wrapped in a `nav` tag. Kindly update that to meet the requirement. Always see requirements as  client needs you have to meet 😄

File: index.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                            src="assets/images/linkedin.svg" alt="linkedin icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/medium.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/github.svg" alt="github icon"></a></li>
+                <li class="heading__icons__li"><a href="#" class="heading__icons__link link"><img
+                            src="assets/images/angellist.svg" alt="angellist icon"></a></li>
+            </ul>
+        </div>
+    </section>
```
Comment (iambenkis) [inline index.html]: Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
Make sure the attributes such as `font`, `colors`, `images`,` text`, `margins`, `background-image` etc are the same 😄 
The next Code reviewer will assess your work according to the update of your work, kindly consider the design in detail 

| Your design | Figma design |
| :---: | :---: |
|Capture | Capture 2 |

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @yemidada 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @yemidada 👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlights
✔ Awesome design 👍 
✔ No linter errors 👍 
✔ Descriptive PR 👍 
✔ The README file is professional and has all the relevant headers 🎆
✔ Made suggested changes👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**